modules/virt-installing-fusion-access-operator.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * virt/fusion_access_SAN/install-configure-fusion-access-san.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-fusion-access-operator_{context}"]
+= Installing the {IBMFusionSAN} Operator
+
+Install the {IBMFusionSAN} Operator from the *OperatorHub* in the {product-title} web console. Once you have installed the Operator, create the `FusionAccess` custom resource (CR) to complete the installation.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have a working registry enabled.
+
+.Procedure
+
+. In the {product-title} web console, navigate to *Operators* -> *OperatorHub*.
+
+. In the *Filter by keyword* field, type `IBM fusion for SAN`.
+
+. Select the *{IBMFusionSAN}* tile and click *Install*.
+
+. On the *Install Operator* page, use the default selections for *Update Channel*, *Version*, and *Installation mode*.
+
+. Verify that *Operator recommended Namespace* is selected for *Installed Namespace*. This installs the Operator in the `ibm-fusion-access` namespace.
++
+If this namespace does not yet exist, it is automatically created.
+
+. Verify that *Automatic* is selected for *Update Approval*. *Automatic* is the default value, and enables automatic updates when a new z-stream release is available.
+
+. Click *Install*.
++
+Once the {IBMFusionSAN} Operator installation is successful, a prompt appears to create  a `FusionAccess` custom resource (CR).
+
+. Click *Create FusionAccess*.
+
+. On the *Create FusionAccess* page, enter the object *Name*.
+
+. You can choose to add *Labels* if they are relevant.
+
+. Verify that the *Ibm Cnsa Version* is correct.
```

Comment:
`Ibm Cnsa Version` was changed to `IBM Storage Scale Version`

Reply:
👍  The full name of the containerized scale product is `IBM Storage Scale container native`.